app/views/api_docs/vendor_api_docs/reference/draft.html.erb

```diff
@@ -128,6 +132,29 @@ We will also support requesting a specific minor version via the URL. For exampl
   If an application is withdrawn at a candidate's request the field <code>withdrawn_or_declined_for_candidate</code> will be set to true to determine the difference between a normal withdrawal and one at a candidate's request. Also either the <code>offer_declined_at</code> field or the <code>date</code> field, inside the existing <%= govuk_link_to 'Withdrawal Object', '#withdrawal-object' %>, will be populated with the timestamp the application was withdrawn or declined.
 </p>
 
+<p class="govuk-body">Only applications in the <code>offer</code> state will transition to the <code>declined</code> state.</p>
+
+<p class="govuk-body">Applications in the following states will transition to the <code>withdrawn</code> state:</p>
+<ul class="govuk-list">
```

Comment:
This (and the below) can go out of date — could we generate this from the state machine?